docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+[//]: # (These are comments and are used for reference and will not show up in the PR)
+
+User Requirement:
```

Comment:
Might be a good idea to change this to "Type of Issue". Types of issues could be like User Requirements, Maintenance, Defects like:

`Type of Issue: User Requirement (xx.xx.xx)`
`Type of Issue: Defect`